shell/browser/api/electron_api_base_window.cc

```diff
@@ -1271,6 +1275,7 @@ void BaseWindow::BuildPrototype(v8::Isolate* isolate,
       .SetMethod("isVisibleOnAllWorkspaces",
                  &BaseWindow::IsVisibleOnAllWorkspaces)
 #if defined(OS_MAC)
+      .SetMethod("getAlwaysOnTopLevel", &BaseWindow::GetAlwaysOnTopLevel)
```

Comment:
let's name it `_getAlwaysOnTopLevel` to make it a bit obvious that it's "internal"